src/EnergyPlus/FluidProperties.cc

```diff
@@ -119,75 +119,31 @@ namespace FluidProperties {
     std::string const Steam("Steam");
     std::string const EthyleneGlycol("EthyleneGlycol");
     std::string const PropyleneGlycol("PropyleneGlycol");
-    int const EthyleneGlycolIndex(-2);
-    int const PropyleneGlycolIndex(-1);
-    int const iRefrig(1);
-    int const iGlycol(1);
 
     // DERIVED TYPE DEFINITIONS
 
     // INTERFACE BLOCK SPECIFICATIONS
     // na
 
     // MODULE VARIABLE DECLARATIONS
-    bool GetInput(true);      // Used to get the input once only
-    int NumOfRefrigerants(0); // Total number of refrigerants input by user
-    int NumOfGlycols(0);      // Total number of glycols input by user
-    bool DebugReportGlycols(false);
-    bool DebugReportRefrigerants(false);
-    int GlycolErrorLimitTest(1);      // how many times error is printed with details before recurring called
-    int RefrigerantErrorLimitTest(1); // how many times error is printed with details before recurring called
-    Array1D_bool RefrigUsed;
-    Array1D_bool GlycolUsed;
-    int FluidIndex_Water(0);
-    int FluidIndex_EthyleneGlycol(0);
-    int FluidIndex_PropoleneGlycol(0);
 
-#ifdef EP_cache_GlycolSpecificHeat
-    int const t_sh_cache_size(1024 * 1024);
-    int const t_sh_precision_bits(24);
-    Int64 const t_sh_cache_mask(t_sh_cache_size - 1);
-#endif
 
     // ACCESSIBLE SPECIFICATIONS OF MODULE SUBROUTINES OR FUNCTONS:
 
     // Object Data
-    Array1D<FluidPropsRefrigerantData> RefrigData;
-    Array1D<FluidPropsRefrigErrors> RefrigErrorTracking;
-    Array1D<FluidPropsGlycolRawData> GlyRawData;
-    Array1D<FluidPropsGlycolData> GlycolData;
-    Array1D<FluidPropsGlycolErrors> GlycolErrorTracking;
 
 #ifdef EP_cache_GlycolSpecificHeat
     Array1D<cached_tsh> cached_t_sh; // DIMENSION(t_sh_cache_size)
 #endif
-
-    // Data Initializer Forward Declarations
+        // Data Initializer Forward Declarations
     // See GetFluidPropertiesData "SUBROUTINE LOCAL DATA" for actual data.
 
-    void clear_state()
-    {
-        GetInput = true;
-        NumOfRefrigerants = 0;
-        NumOfGlycols = 0;
-        DebugReportGlycols = false;
-        DebugReportRefrigerants = false;
-        GlycolErrorLimitTest = 1;
-        RefrigerantErrorLimitTest = 1;
-        RefrigUsed.deallocate();
-        GlycolUsed.deallocate();
-        FluidIndex_Water = 0;
-        FluidIndex_EthyleneGlycol = 0;
-        FluidIndex_PropoleneGlycol = 0;
-        RefrigData.deallocate();
-        RefrigErrorTracking.deallocate();
-        GlyRawData.deallocate();
-        GlycolData.deallocate();
-        GlycolErrorTracking.deallocate();
+void clear_state()
+{
 #ifdef EP_cache_GlycolSpecificHeat
-        cached_t_sh.deallocate();
+    cached_t_sh.deallocate();
```

Comment:
Honestly, I'm thinking that we just don't need to clear this.  This is like one of the only things I can think of in E+ that we can simply leave in place.